pysenteishon/static/js/pysenteishon.js

```diff
@@ -145,8 +148,25 @@ function NetworkInfo ($el) {
 }
 
 
+function Swipe ($el) {
+  this.$el = $el;
+  this.doDoing = false;
+
+  this.do = function(key) {press(key)};
+
+  this.$el.swipe({
+    swipe: function(event, direction, distance, duration, fingerCount, fingerData) {
+      // FIXME: "this.do" doesn't work here and it should :)
+      press(direction);
```

Comment:
@humitos this is rebinded by the swipe plugin , so is referencing a different this

you can change the default this by binding it to the function 

like 

swipe: function() { //code }.bind(this)

or you can use a global alias for it